samples/src/main/kotlin/com/simplemobiletools/commons/samples/activities/MainActivity.kt

```diff
@@ -2,55 +2,178 @@ package com.simplemobiletools.commons.samples.activities
 import com.simplemobiletools.commons.samples.R
-import com.simplemobiletools.commons.samples.databinding.ActivityMainBinding
-
-class MainActivity : BaseSimpleActivity() {
-    override fun getAppLauncherName() = getString(R.string.smtco_app_name)
-
-    override fun getAppIconIDs(): ArrayList<Int> {
-        val ids = ArrayList<Int>()
-        ids.add(R.mipmap.commons_launcher)
-        return ids
-    }
-
-    private val binding by viewBinding(ActivityMainBinding::inflate)
+import kotlinx.collections.immutable.toImmutableList
 
+class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(binding.root)
-        appLaunched(BuildConfig.APPLICATION_ID)
 
-        updateMaterialActivityViews(binding.mainCoordinator, binding.mainHolder, useTransparentNavigation = true, useTopSearchMenu = false)
-        setupMaterialScrollListener(binding.mainNestedScrollview, binding.mainToolbar)
+        enableEdgeToEdgeSimple()
+        setContent {
+            AppThemeSurface {
+                SettingsLazyScaffold(
+                    customTopBar = { scrolledColor: Color, _: MutableInteractionSource, scrollBehavior: TopAppBarScrollBehavior, statusBarColor: Int, colorTransitionFraction: Float, contrastColor: Color ->
+                        TopAppBar(
+                            title = {},
+                            actions = {
+                                val actionMenus = remember {
+                                    val about = ActionItem(
+                                        com.simplemobiletools.commons.R.string.about,
+                                        icon = Icons.Outlined.Info,
+                                        doAction = ::launchAbout,
+                                        overflowMode = OverflowMode.NEVER_OVERFLOW
+                                    )
+                                    val moreApps =
+                                        ActionItem(
+                                            com.simplemobiletools.commons.R.string.more_apps_from_us,
+                                            doAction = ::launchMoreAppsFromUsIntent,
+                                            overflowMode = OverflowMode.ALWAYS_OVERFLOW
+                                        )
+                                    listOf(about, moreApps).toImmutableList()
+                                }
+                                var isMenuVisible by remember { mutableStateOf(false) }
+                                ActionMenu(
+                                    items = actionMenus,
+                                    numIcons = 2,
+                                    isMenuVisible = isMenuVisible,
+                                    onMenuToggle = { isMenuVisible = it },
+                                    iconsColor = scrolledColor
+                                )
```

Comment:
Please extract in a function for better readability

Reply:
I will also extract the screen then, to make it a bit more similar to how it should be done in actual apps.